model/src/main/scala/org/broadinstitute/dsde/rawls/model/ErrorReportable.scala

```diff
@@ -12,11 +11,11 @@ trait ErrorReportable {
     throwable match {
       case gjre: GoogleJsonResponseException =>
         val statusCode = StatusCodes.getForKey(gjre.getStatusCode)
-        ErrorReport(ErrorReport.message(gjre), statusCode, ErrorReport.causes(gjre), Seq.empty, Option(gjre.getClass))(errorReportSource)
+        ErrorReport(ErrorReport.message(gjre), statusCode, ErrorReport.causes(gjre)(errorReportSource), Seq.empty, Option(gjre.getClass))(errorReportSource)
```

Comment:
I would need a special test for this, that test case does not have an exception that lists causes